ci-operator/step-registry/storage/conf/csi-azure-file-nfs/storage-conf-csi-azure-file-nfs-commands.sh

```diff
@@ -3,6 +3,33 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+export STORAGECLASS_LOCATION=${SHARED_DIR}/azurefile-nfs-sc.yaml
+
+# Create StorageClass
+echo "Creating a StorageClass"
+cat <<EOF >>$STORAGECLASS_LOCATION
+# Taken from https://github.com/kubernetes-sigs/azurefile-csi-driver/blob/master/deploy/example/storageclass-azurefile-nfs.yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: azurefile-csi-nfs
+provisioner: file.csi.azure.com
+parameters:
+  protocol: nfs
+  skuName: Premium_LRS
```

Comment:
`assets/storageclass_nfs.yaml` from https://github.com/openshift/azure-file-csi-driver-operator doesn't have `skuName` and the test (`ci/prow/e2e-azure-file-nfs-csi`) is more-or-less happy w/o it. Do we consider the lack of `skuName` there as a bug and fix it here or there are other reasons for adding it?

Reply:
See a few lines above, this SC was taken from upstream, which is more or less a recommendation from MS.

The original SC you pointed above was also taken from upstream, but at the time it didn't have `skuName`, nor the `nsconnect` mount option.